spacewalk/certs-tools/rhn_bootstrap.py

```diff
@@ -41,10 +41,9 @@
 from spacewalk.common.rhnConfig import CFG, initCFG
 from .client_config_update import readConfigFile
 from .rhn_bootstrap_strings import \
-    getHeader, getConfigFilesSh, getUp2dateScriptsSh, getGPGKeyImportSh, \
-    getCorpCACertSh, getRegistrationSh, getUp2dateTheBoxSh, \
-    getAllowConfigManagement, getAllowRemoteCommands, \
-    getRegistrationStackSh, getRegistrationSaltSh, removeTLSCertificate
+    getHeader, getGPGKeyImportSh, \
+    getCorpCACertSh, getRegistrationStackSh, \
+    getRegistrationSaltSh, removeTLSCertificate
 from .sslToolConfig import CA_CRT_NAME, CA_CRT_RPM_NAME
```

Comment:
`CA_CRT_RPM_NAME` is not used. While it wasn't removed in this PR, since you touch the surrounding code it could be a good time to remove its import as well.

Reply:
removed